New issue template: Edit this page. (#6471)
DataDog/documentation: .github/ISSUE_TEMPLATE/edit_this_page.md

---
name: Edit this page

> about: Many thanks for your feedback but the page you are trying to edit is not available publicly. Please fill out the information below and Datadog Documentation team will take care of updating the corresponding documentation page.

Reviewer: ```suggestion
about: Thanks for your feedback, but the page you are trying to edit is not available publicly. Fill out the information below, and Datadog's Documentation team will take care of updating the corresponding documentation page.
```

---

> **Documentation page to update**:

Reviewer: are we asking for them to put the URL? let's ask explicitly for a URL

Author: ```suggestion
**Documentation page URL to update**:
```

> **Describtion of the update to make**:

Reviewer: ```suggestion
**Description of the update**:
```
